src/WebJobs.Extensions.CosmosDB/azure-pipelines.yml

```diff
@@ -0,0 +1,40 @@
+parameters: # user-configurable parameters
+  - name: skip_assembly_signing
+    displayName: Skip Assembly Signing
+    type: boolean
+    default: false
+
+pr:
+  branches:
+    include:
+    - main
+    - dev
+  paths:
+    exclude: # don't trigger builds for file changes matching these paths
+    - src/WebJobs.Extensions.Http/*
+    - src/WebJobs.Extensions.MobileApps/*
+    - src/WebJobs.Extensions.SendGrid/*
+    - src/WebJobs.Extensions.Timers.Storage/*
+    - src/WebJobs.Extensions.Twilio/*
+
+trigger:
+  branches:
+    include:
+    - main
+    - dev
+  paths:
+    exclude: # don't trigger builds for file changes matching these paths
+    - test/WebJobs.Extensions.Http.Tests/*
```

Comment:
Should this also include Cosmos DB tests?

Reply:
The inclusion should be implicit - `exclude` acts as 'everything except what's listed.'